services-js/access-boston/src/server/services/IdentityIq.ts

```diff
@@ -7,6 +7,8 @@ const LAUNCHED_WORKFLOW_SCHEMA =
   'urn:ietf:params:scim:schemas:sailpoint:1.0:LaunchedWorkflow';
 
 const CHANGE_PASSWORD_WORKFLOW = 'COB-RESTAPI-Workflow-ChangePassword';
+// TODO(finh): Update when the final workflow is done
```

Comment:
I assume this is still in progress?

Reply:
Yeah, waiting on IAM team to get that eventually, and then we'll change it on our side.